array.c

```diff
@@ -3097,20 +3097,21 @@ rb_ary_to_s(VALUE ary)
  *  call-seq:
  *    to_a -> self or new_array
  *
- *  Returns +self+ if <tt>self.instance_of?(Array)</tt>:
+ *  When +self+ is an instance of \Array, returns +self+:
  *    a = [:foo, 'bar', 2]
+ *    a.instance_of?(Array) # => true
  *    a1 = a.to_a
- *    a1 # => [:foo, "bar", 2]
  *    a1.equal?(a) # => true # Returned self
  *
- *  If +self+ is a subclass of \Array, returns the new \Array
- *  formed by converting +self+ to an \Array:
+ *  Otherwise, when <tt>self.kind_of?(Array)</tt>,
```

Comment:
`,  when <tt>self.kind_of?(Array)</tt>`  can be removed as it is redundant.  All cases where the method can be called where self is not an instance of Array, self is a kind of Array.

Reply:
Removed.

Reply:
This says removed, but the changes haven't been made.  Do you need to push a commit?

Reply:
Duh, yes. When I was in the Army (back at the Dawn of Time), the instructors used to say, "Take your right ear in your right hand, and your left ear in your left hand, and pull you head out of ...."
Now pushed.